### PR TITLE
Highlight active tab

### PR DESCRIPTION
**Pull request by TheZoker**
Highlight active tab by using the `location` param, so the user knows, where he currently is.

### CODE REVIEW

File: custom_components/hacs/frontend/templates/base.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {% set progressbar = 'block' if hacs.system.status.background_task else 'none' %}
+{% set overviewActive = 'active' if location == 'overview' else '' %}
+{% set storeActive = 'active' if location == 'store' or location == 'repository' else '' %}
+{% set settingsActive = 'active' if location == 'settings' else '' %}
```
Comment (ludeeus) [inline custom_components/hacs/frontend/templates/base.html]: This is not needed.

File: custom_components/hacs/frontend/templates/base.html [hunk 2]
```diff
@@ -15,9 +18,9 @@
     <nav class="nav-extended hacs-nav">
       <div class="nav-content">
         <ul class="right tabs tabs-transparent">
-          <li class="tab"><a href="/hacsweb/{{ hacs.token }}/overview?timestamp={{ timestamp }}">overview</a></li>
-          <li class="tab"><a href="/hacsweb/{{ hacs.token }}/store?timestamp={{ timestamp }}">store</a></li>
-          <li class="tab right"><a href="/hacsweb/{{ hacs.token }}/settings?timestamp={{ timestamp }}">settings</a></li>
+          <li class="tab {{ overviewActive }}"><a href="/hacsweb/{{ hacs.token }}/overview?timestamp={{ timestamp }}">Overview</a></li>
```
Comment (ludeeus) [inline custom_components/hacs/frontend/templates/base.html]: simplify move the logic down here eg:

```
<li class="tab {{ 'active' if location == 'overview' }}">
```

File: custom_components/hacs/http.py [hunk 3]
```diff
@@ -214,7 +214,7 @@ class Repository(HacsWebResponse):
             repository.status.new = False
             self.data.write()
 
-        render = self.render("repository", repository=repository, message=message)
+        render = self.render("repository", "repository", repository=repository, message=message)
```
Comment (ludeeus) [inline custom_components/hacs/http.py]: You open the repository view from all 3 tabs, when in a repository view there should not be any active tab (unless you add logic so it remembers it revert this change

**Comment by TheZoker** [comment]
Resolved all requests

**Comment by ludeeus** [comment]
This was much cleaner, thank you 🏅
